app/js/services/trampolineService.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+angular.module('ddsCommon').factory('TrampolineService', function($localStorage) {
```

Comment:
I think we could avoid the ["smurf naming convention"](https://blog.codinghorror.com/new-programming-jargon/)  😀

A cleaner interface would be: 
- `set(name, value = null)` → accepts a hash or a name + value
- `get(name)` → returns the whole hash, or just the key if provided. When a key is retrieved, it is removed 
- `clear()` → clears everything
- `has(name)` → allows to check if a key exists, without deleting it 
- `keys()` → returns the keys, without deleting them

Reply:
Inspired by [Symfony Flash Messages](https://symfony.com/doc/current/components/http_foundation/sessions.html#flash-messages)  😎